apps/test/unit/applab/designModeTest.js

```diff
@@ -134,6 +135,10 @@ describe('setProperty and read Property', () => {
       expect(text_area.innerHTML).to.equal('Lambda Mu');
       expect(dropdown.value).to.equal('Eta Theta');
     });
+    it('Uses the asset timestamp in the source path for pictures', () => {
+      designMode.updateProperty(picture, 'picture', 'picture.jpg', 123456);
+      expect(picture.src).to.contain('picture.jpg?t=123456');
+    });
```

Comment:
Is it worth adding a test to verify the correct behavior if updateProperty is called with 'picture.jpg' more than once?